akasha-vladan-main/css/style.scss

```diff
@@ -0,0 +1,76 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap');
+
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: 'Open Sans', sans-serif;
+
+}
+
+.title-wrap {
+    background-color: black;
+    padding-top: 15px;
+    padding-bottom: 15px;
+}
+
+
+#title{
+    color: white;
+    text-align: center;
+}
+
+#img-div{
+    width: 400px;
+    max-width: 100%;
+    margin: auto;
+    margin-top: 20px;
+    padding: 0px 30px;
+    #image{
+        max-width: 100%;
+        display: block;
+        border-radius: 50%;
+        box-shadow: 0px 10px 15px rgba(0, 0, 0, 0.8);
+        margin-bottom: 20px;
+        &:hover{
+            transform: scale(1.03);
+            transition: .3s;
+        }
+    }
+    #img-caption{
+        text-align: center;
+        margin-top: 10px;
+        font-size: 16px;
+    }
+}
+
```

Comment:
kindly apply best css practices by adding an indentation of 2 space when writing the css rule 
so as not to have linter errors

Reply:
- [ ] I think you should always include an explanation, a "why", for each requested change. 

For example:

This is a well explained request: _Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible_

This request lacks a "why": _Why don't you use an article tag here instead of a div?_